modules/custom/openy_activity_finder/src/Plugin/Block/ActivityFinderBlock.php

```diff
@@ -30,6 +30,28 @@ public function build() {
       $alias = \Drupal::service('path.alias_manager')->getAliasByPath('/node/' . $node->id());
     }
 
+    $locationsMapping = [];
+    if ($config->get('backend' == 'openy_daxko2.openy_activity_finder_backend')) {
+      $openy_daxko2_config = \Drupal::service('config.factory')->get('openy_daxko2.settings');
+      if (!empty($openy_daxko2_config->get('locations'))) {
+        $nids = \Drupal::entityQuery('node')
+          ->condition('type', ['branch', 'camp', 'facility'], 'IN')
+          ->condition('status', 1)
+          ->sort('title', 'ASC')
+          ->execute();
+        $locations = \Drupal::service('entity_type.manager')->getStorage('node')->loadMultiple($nids);
```

Comment:
add DI, please

Reply:
@atiupa , this could be replaced to Node::loadMultiple($nids)

Reply:
fixed